config/srasConfig.json

```diff
@@ -1,15 +1,11 @@
 {
-    "ignore": [
-        "js",
-        "ts",
-        "css",
-        "html",
-        "json"
-    ],
     "unpermitted": [
         "wav"
     ],
     "rules": {
+        "js": {
```

Comment:
Might be a good idea to nest 'js' in a 'scripts' object similar to 'images' and 'audio'. This will make adding new script file types ('json', 'html', etc) easier as it will only required a config addition and not a code addition.

Reply:
I struggled with this, since I wouldn't consider JSON nor HTML to be scripts. IMO, if we nested inside of a scripts object we would end up with only one subkey: scripts -> js. However, if we could come up with a better name than `scripts` I think I would be cool with nesting. So, any good name ideas?

Reply:
`code`, `files`, `data`, `other`, `extra`

I was using the term `script` very broadly to cover anything that is code-like.

Reply:
This is fixed now: I've nested JS rules under a `code` key.